### ISSUE TITLE
vacation/out-of-office filter does not send when using outgoing smtp relay

### ISSUE THREAD

**Issue by s4069b** [ddavness/power-mailinabox#66]
Is anyone else experiencing this?
I have Amazon SES setup as a custom smtp for outgoing mail. 
When I setup an out of office auto-reply (using roundcube)... the auto reply emails only trigger if I turn off the smtp relay.

**Comment by ddavness** [comment]
That definitely could look like a bug, but I'll need to investigate this further. It could be an issue with Power Mail-in-a-Box, but it could also be an issue with the relay itself :(

**Comment by s4069b** [comment]
following up on this. 
I tried a regular install of MIAB with Amazon SES setup as a relay host following the the pointers here - https://www.linode.com/docs/guides/postfix-smtp-debian7/ . I realised the changes may not remain after making changes/updates to MIAB. 
With this setup I experienced the same results - ie sieve auto-reply did not work. 
I then edited data/conf/dovecot/dovecot.conf and added this to the bottom
plugin {
  sieve_vacation_send_from_recipient = yes
} 
After restarting dovecot and postfix... auto replies started working through the amazon SES relay host ... as expected. 
I suspect the same would work with Power MIAB too. But I have not tested.
